src/index.js

```diff
@@ -1,51 +1,28 @@
-const addAngularServerlessFiles = require('./helpers/addAngularServerlessFiles')
 const getAngularJson = require('./helpers/getAngularJson')
 const getAngularRoot = require('./helpers/getAngularRoot')
-const setUpBuilderFunction = require('./helpers/setUpBuilderFunction')
-const setUpFunctionsConfig = require('./helpers/setUpFunctionsConfig')
-const setUpRedirects = require('./helpers/setUpRedirects')
-const updateAngularJson = require('./helpers/updateAngularJson')
-const validateAngularUniversalUsage = require('./helpers/validateAngularUniversalUsage')
-const validateNetlifyConfig = require('./helpers/validateNetlifyConfig')
+const setUpEdgeFunction = require('./helpers/setUpEdgeFunction')
+const validateAngularVersion = require('./helpers/validateAngularVersion')
 
 module.exports = {
-  async onPreBuild({ netlifyConfig, utils }) {
+  async onPreBuild({ utils }) {
     const { failBuild } = utils.build
-
-    validateAngularUniversalUsage({ failBuild })
-
-    const siteRoot = getAngularRoot({ failBuild, netlifyConfig })
-
-    const angularJson = getAngularJson({ failBuild, siteRoot })
-
-    const projectName = angularJson.defaultProject
-
-    updateAngularJson({ angularJson, failBuild, projectName, siteRoot })
-
-    validateNetlifyConfig({ failBuild, netlifyConfig, projectName })
-
-    addAngularServerlessFiles({ projectName, siteRoot })
+    // skipping this for now, something's off with the demo site
```

Comment:
need to change this before we can merge

Reply:
done in https://github.com/netlify/netlify-plugin-angular-universal/pull/67/commits/86e2d91360b5b2c5294aa55023af7830d0602aae